src/main/java/org/broadinstitute/dsm/jobs/UPSTrackingJob.java

```diff
@@ -235,6 +232,15 @@ else if (statusType.equals(DELIVERY) && !(DELIVERY.equals(oldType))) {
         }
     }
 
+    /**
+     * Determines whether or not a trigger should be sent to
+     * study-server to respond to kit being sent to participant
+     */
+    private static boolean shouldTriggerKitDeliveryEvent(String currentStatus, String previousStatus) {
```

Comment:
Intent here is that if the kit is newly in the `D` or `I` category, an event should be sent.  But state changes from `I` to `D` should not trigger an update.  Only a change from something other than `D` or `I` into `D` or `I` state should trigger the event.

Reply:
If kit is in `in_transit`, how long does it typically take for it to be physically delivered in the real world? The "delivered" event will trigger the symptom surveys, one of which is the "longitudinal" covid survey. That one has an expiration time of 7 days, after which it becomes read-only. I imagine it might only take 1 or so for that state transition, so users should have plenty of time?